packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/components/TextQuestion.js

```diff
@@ -7,44 +7,37 @@ import { UID } from "react-uid";
 import { theme } from "@cdt/ui";
 import styled from "styled-components";
 
-function TextQuestion({ name, label, inputType = "text", size }) {
-  return (
-    <UID>
-      {id => (
-        <>
-          <QuestionLabel htmlFor={id}>{label}</QuestionLabel>
-          <QuestionWrapper>
-            <Field
-              name={name}
-              subscribe={{ touched: true, invalid: true }}
-              validate={required}
-              render={({ input, meta: { touched, invalid } }) => (
+const TextQuestion = ({ name, label, inputType = "text", size }) => (
```

Comment:
function is OK :) so I propose to revert that too

Reply:
👌